### PR TITLE
Missing class ul tag

### PR DESCRIPTION
**Pull request by flash1452**
Pull Request for Issue #8945  .

#### Summary of Changes
Introduced a new class in the `<ul>` tag of file mod_tags_popular and mod_tags_similar files.
#### Testing Instructions
The tag was a simple `<ul>` tag 
Now its: `<ul class="tagspopular">`

### CODE REVIEW

File: modules/mod_tags_similar/tmpl/default.php [hunk 1]
```diff
@@ -13,7 +13,8 @@
 <?php JLoader::register('TagsHelperRoute', JPATH_BASE . '/components/com_tags/helpers/route.php'); ?>
 <div class="tagssimilar<?php echo $moduleclass_sfx; ?>">
 <?php if ($list) : ?>
-	<ul>
+
+	<ul class="tagspopular<?php echo $moduleclass_sfx; ?>">
```
Comment (wilsonge) [inline modules/mod_tags_similar/tmpl/default.php]: Please remove the module class suffix - it's done very inconsistently (and we can't fix it in J3 for b/c) - but the "correct" thing is for it to be placed in the module "wrapper" in the templates modules.php file e.g https://github.com/joomla/joomla-cms/blob/staging/templates/protostar/html/modules.php#L47 not inside the module

Comment (flash1452) [inline modules/mod_tags_similar/tmpl/default.php]: One more thing I had like to ask ,in mod_tags_similar/tmpl/default.php,the class in ul tag should be tagsimilar I suppose?

**Comment by Twincarb** [comment]
I have tested this item :white_check_mark: successfully on 10d956f699a45bf3be38ccbae1d5081f8d8d450d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9388">issues.joomla.org/joomla-cms/9388</a>.</sub>

File: modules/mod_tags_similar/tmpl/default.php (hunk 1 above)
Comment (brianteeman) [inline modules/mod_tags_similar/tmpl/default.php]: @wilsonge thats the whole point of this PR to bring this module in line with all the others

Comment (brianteeman) [inline modules/mod_tags_similar/tmpl/default.php]: @flash1452 yes thats correct

**Comment by Twincarb** [comment]
I have tested this item :white_check_mark: successfully on 95b25c7152eede7079ce0a795bfedcb527ba4249<br /><br />Tested both mod_tags_popular and mod_tags_similar both showing new ul class<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9388">issues.joomla.org/joomla-cms/9388</a>.</sub>

**Comment by flash1452** [comment]
@brianteeman Please review my PR.

**Comment by ghazal** [comment]
I have tested this item :white_check_mark: successfully on 95b25c7152eede7079ce0a795bfedcb527ba4249<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9388">issues.joomla.org/joomla-cms/9388</a>.</sub>

File: modules/mod_tags_popular/tmpl/default.php [hunk 2]
```diff
@@ -15,7 +15,8 @@
 <?php if (!count($list)) : ?>
 	<div class="alert alert-no-items"><?php echo JText::_('MOD_TAGS_POPULAR_NO_ITEMS_FOUND'); ?></div>
 <?php else : ?>
-	<ul>
+
```
Comment (coolbung) [inline modules/mod_tags_popular/tmpl/default.php]: Is this extra line needed ?

File: modules/mod_tags_similar/tmpl/default.php [hunk 3]
```diff
@@ -13,7 +13,8 @@
 <?php JLoader::register('TagsHelperRoute', JPATH_BASE . '/components/com_tags/helpers/route.php'); ?>
 <div class="tagssimilar<?php echo $moduleclass_sfx; ?>">
 <?php if ($list) : ?>
-	<ul>
+
```
Comment (coolbung) [inline modules/mod_tags_similar/tmpl/default.php]: Same here..

**Comment by lunalars** [comment]
As there already is a class on the surrounding div (tagspopular / tagssimilar) you could address the ul like this: .tagssimilar > ul in your css. So I think there is no need for a class on the ul - keeps code cleaner.

**Comment by faynt0** [comment]
I have tested this item :white_check_mark: successfully on 477414767c3f1891e3346d4592241a0f83e0d1b8<br /><br />Patch does add mentioned class to `<ul>` of popular tags.

But as @lunalars mentioned, the parent div already has a appropriated class to select the tags.
Post-patch there will be a parent `<div class="tagspopular">` with a child `<ul class="tagspopular">` which seems messy.

Hosted by @icampus
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9388">issues.joomla.org/joomla-cms/9388</a>.</sub>

**Comment by abuechert** [comment]
I have tested this item :white_check_mark: successfully on 477414767c3f1891e3346d4592241a0f83e0d1b8<br /><br />tested @icampus

patch does what it says. adds a <code>class=tagspopular</code> to the <code><ul></code> tag of popular tags<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9388">issues.joomla.org/joomla-cms/9388</a>.</sub>

**Comment by roland-d** [comment]
Ready to commit as we have 2 successful tests.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9388">issues.joomla.org/joomla-cms/9388</a>.</sub>

**Comment by wilsonge** [comment]
Removing RTC label - the div and ul element shouldn't have the same class as has been pointed out.

**Comment by roland-d** [comment]
@wilsonge Oh hit me, I missed that :-|

**Comment by roland-d** [comment]
We can't fix this in a backwards compatible way because you can't remove the class on the div as it may break existing sites and we can't add the class to the ul as it would be a duplicate class. A real catch-22, only way I see this fixed is in Joomla 4. Thanks for your effort.

**Pull request closed without merging**